ugvc/mrd/srsnv_training_utils.py

```diff
@@ -185,28 +189,23 @@ def prepare_featuremap_for_model(
     if random_seed is not None:
         np.random.seed(random_seed)
     if balanced_sampling_info_fields:
-        # TODO test this code and add a test, I wrote an implementation but didn't even get to run it
-        logger.warning(
-            "Data motif balancing not tested, use at your own risk"
-            f" (balanced_sampling_info_fields={balanced_sampling_info_fields})"
-        )
         # count the number of entries with each combination of info fields to balance by
         balanced_sampling_info_fields_counter = defaultdict(int)
         with pysam.VariantFile(intersect_featuremap_vcf) as fmap:
```

Comment:
Wouldn't it be better to count while in the previous general counting loop, for efficiency reasons.

Reply:
You're right, fixed.